arkane/mainTest.py

```diff
@@ -55,6 +56,11 @@ def setUpClass(cls):
         cls.base_path = os.path.join(os.path.dirname(os.path.dirname(rmgpy.__file__)), 'examples', 'arkane')
         cls.failed = []
         cls.example_types = ['species', 'reactions', 'explorer', 'networks']
+        ch2ooh_path = os.path.join(cls.base_path,'species','CH2CHOOH','CH2CHOOHscans')
```

Comment:
space between arguments? (below as well)

Reply:
done

Reply:
Can you take a second look?
It's minor, but I don't think its resolved.
It should be `os.path.join(cls.base_path, 'species', 'CH2CHOOH', 'CH2CHOOHscans')`,
but I still see `os.path.join(cls.base_path,'species','CH2CHOOH','CH2CHOOHscans')`.

same thing below, and possibly other places in this PR